data/seeds/action.js

```diff
@@ -0,0 +1,22 @@
+
+exports.seed = function(knex, Promise) {
```

Comment:
The only suggestion I can make here is about the name of the file, remember that seedings doesn't have a timestamp in the name and we need to somehow make sure that all the seedings are running in the order we want them to.